### PR TITLE
BUG: magic factor for Netapp LUN checks are missing

### PR DESCRIPTION
**Pull request by Yogibaer75**
The magic factor settings for this "filesystem" check was missing inside the WATO options.

### CODE REVIEW

**Comment by jherbel** [comment]
Hi Andreas,
thank you for this pull request. Before we can take a closer look, please make sure that all tests working. See https://github.com/tribe29/checkmk/blob/master/CONTRIBUTING.md for more information, also on how to run the tests locally.
Best
Jörg

File: cmk/gui/plugins/wato/check_parameters/netapp_luns.py [hunk 1]
```diff
@@ -57,6 +58,32 @@ def _parameter_valuespec_netapp_luns():
                      )
                  ],
              )),
+            ("magic",
```
Comment (mo-ki) [inline cmk/gui/plugins/wato/check_parameters/netapp_luns.py:61]: This duplicates code. At least we should make `_filesystem_magic_elements` in `cmk/gui/plugins/wato/check_parameters/filesystem_utils.py` "public" (strip the leading underscore), and use it here. I think we could consolidate even more, but at least we should not add more duplicate code....

**Review by mo-ki — CHANGES_REQUESTED** [review]

File: cmk/gui/plugins/wato/check_parameters/netapp_luns.py (hunk 1 above)
Comment (Yogibaer75) [reply cmk/gui/plugins/wato/check_parameters/netapp_luns.py:61]: Hi Moritz - nightshift today? :)
inside the filesystem_utils.py the part size_trend_elements is already public. Then the same for magic_elements would be easiest way i think.

**Comment by mo-ki** [comment]
My colleague did a lot of clean ups and refactorings. Thanks to her, this is now easily implemented. I'll do that shortly.

**Pull request closed without merging**